app/views/IncomeBeforeTaxView.scala.html

```diff
@@ -17,106 +17,123 @@
-@import views.html.includes.SaMainView
-@import views.html.includes.TaxsBreadcrumbView
-@import views.html.helpers.BackLinkView
 @import views.html.includes.CurrencyAmountView
-
-@this(saMainView: SaMainView, taxsBreadcrumbView: TaxsBreadcrumbView, backLinkView: BackLinkView, currencyAmountView: CurrencyAmountView)
-
-@(viewModel: view_models.IncomeBeforeTax, actingAsAttorney: Option[ActingAsAttorneyFor] = None)(implicit request: AuthenticatedRequest[_], messages: Messages, formPartialRetriever: FormPartialRetriever, appConfig: ApplicationConfig)
-
-@headerContent = {
-  @taxsBreadcrumbView(List("breadcrumbs.taxs.select_tax_year", "breadcrumbs.taxs.index", "breadcrumbs.taxs.summary"), "ats.income_before_tax.title", viewModel.taxYear)
-
-  @if(actingAsAttorney.isDefined) {
-    <div class="nested-banner">
-      <p id="agent-banner">@{Messages("generic.acting_for").mkString} @{actingAsAttorney.get.name.getOrElse(Messages("generic.name_not_defined")).split(" ").map(_.capitalize).mkString(" ")}.</p>
+@import views.html.main
+@import views.html.helpers.HeaderSection
+
+@this(
+    main: main,
+    headerSection: HeaderSection,
+    currencyAmountView: CurrencyAmountView
+)
+
+@(  viewModel: view_models.IncomeBeforeTax,
+    actingAsAttorney: Option[ActingAsAttorneyFor] = None
+    )(
+    implicit request: AuthenticatedRequest[_],
+    messages: Messages,
+    formPartialRetriever: FormPartialRetriever,
+    templateRenderer: uk.gov.hmrc.renderer.TemplateRenderer,
+    appConfig: ApplicationConfig
+)
+
+@main(
+        title = Messages("ats.income_before_tax.html.title") + Messages("generic.to_from", viewModel.taxYearFrom, viewModel.taxYearTo),
+        actingAttorney = actingAsAttorney,
+        articleClasses = Some("full-width")
+    ) {
+
+    @headerSection(
+        pageHeading = Messages("ats.income_before_tax.title"),
+        name = viewModel.forename + ' ' + viewModel.surname,
+        utr = viewModel.utr,
+        taxYear = viewModel.taxYear,
+        actingAsAttorney = actingAsAttorney,
+        breadcrumbLinks = List(
+            "breadcrumbs.taxs.select_tax_year",
+            "breadcrumbs.taxs.index",
+            "breadcrumbs.taxs.summary"
+        ),
+        currentBreadcrumbLink = "ats.income_before_tax.title"
+    )
+
+    <div class="main-content">
+
+        <h1>
+            <span class="gray heading-medium block-heading">
+              @Html(Messages("generic.tax_year_heading", viewModel.taxYearFrom, viewModel.taxYearTo))
+            </span>
+          @Html(Messages("ats.income_before_tax.title"))
+        </h1>
+        <p class="lede">@Html(Messages("ats.income_before_tax.intro"))                    </p>
+
+        <table>
+          <thead>
+            <tr>
+              <th>@Html(Messages("ats.income_before_tax.table_header"))</th>
+              <th>@Html(Messages("generic.amount_pounds"))</th>
+            </tr>
+          </thead>
+
+          <tbody id="income-before-tax-table">
```

Comment:
more of a question than something to action but couldn't this be rewritten almost using a map/case statments to pass the table rows statement and just pass variables to the first td message, id and which view model to save from having to rewrite each table row

Reply:
Resolving as ticket is just covering template upgrade